### ISSUE TITLE
[A11y][ScreenReader]Package Manager tab on Package details reads as both selected and expanded

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9418]
Url: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1744434

Environment Details:
App name: NuGet Service
#URL: [https://www.nuget.org](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.nuget.org%2f&data=05%7c01%7cv-nitipi%40microsoft.com%7c9a84307e7aa744a4b24208dafa5ca2fe%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638097573052590837%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=C8Eab3i6ixAmqsOjG4AxNT36z9WZQiXPbjDDkMcS5ME%3D&reserved=0)

OS and Browser Details:
Edition Windows 11 Enterprise
OS build 22621.1105.
Chrome version:  109.0.5414.120 (Official Build) (64-bit)
Screen Reader: JAWS, NVDA

Repro Steps:
Hit the URL: https://www.nuget.org/
Navigate to "Sign in" control and hit enter.
Navigate to "Sign in with Microsoft" control and hit enter.
Search for a key term (say “json”) from search box.
Click on "Newtonsoft.json" control and hit enter.
Navigate to "PackageReference" control and hit enter.
Observe whether screen reader should announce only one state as 'selected' or not.

Actual Result:
Screen reader is announcing two states as 'selected and expanded' for "package reference" tab item.

Expected Result:
Screen reader should announce only one states as 'selected' for "package reference" tab item.

Notes:
These controls appears to have aria-expanded defined on them for some reason. I am unsure why this is the case, but it seems like removing it will address the issue here. However, I don't know what other issues it might introduce.
